### DISCUSSION TITLE
Possibility to use JSONMatch in decision tables

### DISCUSSION THREAD

**Discussion by DanielVillacanas** [gorules/zen#42] [Q&A]
Hi guys.

I have a question, I want to know if it is possible to use JSONMatch in decision tables, or if it is possible to use an array of JSON objects as input (request) instead of a simple JSON.

Thanks in advance

**Comment by ivanmiletic** [comment]
Hi @DanielVillacanas at the moment we do not support JSONMatch but using arrays inside decision tables is something that we are discussing with one of our clients.

From what it looks like we will be adding this capability but not as a priority.
